lit_gpt/args.py

```diff
@@ -17,12 +17,18 @@ class TrainArgs:
     """Number of samples per data-parallel rank"""
     lr_warmup_epochs: int = 2
     """Number of epochs with learning rate warmup active"""
+    # TODO: some scripts use steps, other iters. Standardize?
```

Comment:
Yes, otherwise it's a bit confusing.
I think there was a question about this from one of the users somewhere last year.